proposals/csharp-9.0/records.md

```diff
@@ -197,6 +197,115 @@ If the containing record is abstract, the synthesized clone method is also abstr
 If the "clone" method is not abstract, it returns the result of a call to a copy constructor. 
 
 
+### Printing members: print and ToString methods
+
+If the record is derived from `object`, the record includes a synthesized "print" method equivalent to a method declared as follows, but with a compiler-reserved name:
```

Comment:
>"print" method equivalent to a method declared as follows, but with a compiler-reserved name [](start = 74, length = 92)

The spec should use the actual name since it is part of the contract, even if the name cannot be used from C#.

Reply:
We didn't do that for the "clone" method. I'm following that model.